src/Concerns/ManagesInvoices.php

```diff
@@ -160,11 +160,18 @@ public function createInvoice(array $options = [])
     {
         $this->assertCustomerExists();
 
+        $stripeCustomer = $this->asStripeCustomer();
+
         $parameters = array_merge([
             'automatic_tax' => $this->automaticTaxPayload(),
             'customer' => $this->stripe_id,
+            'currency' => $stripeCustomer->currency ?? config('cashier.currency'),
         ], $options);
 
+        if (isset($parameters['subscription'])) {
+            unset($parameters['currency']);
+        }
+
```

Comment:
`currency` cannot be combined with `subscription` so we allow `subscription` to take precedence.